src/Security/Member.php

```diff
@@ -447,59 +422,41 @@ public function isPasswordExpired()
     }
 
     /**
-     * Logs this member in
+     * @deprecated Use Security::setCurrentUser() or IdentityStore::logIn()
```

Comment:
You can use @deprecated 4.0..5.0 to declare the deprecation assignment version, as well as the target API removal date.